.github/workflows/compliance-check.yml

```
@@ -132,3 +132,4 @@ jobs:
             echo "Compliance check FAILED"
             exit 1
           fi
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)